packages/story-editor/src/components/canvas/rightClickMenu.js

```diff
@@ -72,6 +111,35 @@ const RightClickMenu = () => {
     };
   }, [ref]);
 
+  const menuButtonRenderer = ({
+    label,
+    shortcut,
+    icon,
+    supportsIcon,
+    SuffixIcon,
+    ...buttonProps
+  }) => (
+    <ContextMenuComponents.MenuButton {...buttonProps}>
+      {supportsIcon && (
+        <LabelWrapper>
+          {icon}
+          {label}
+        </LabelWrapper>
+      )}
+      {!supportsIcon && label}
+      {shortcut && (
+        <ContextMenuComponents.MenuShortcut>
+          {shortcut.display}
+        </ContextMenuComponents.MenuShortcut>
+      )}
+      {SuffixIcon && (
+        <SuffixIconWrapper>
+          <SuffixIcon />
+        </SuffixIconWrapper>
```

Comment:
🤔 The chevron should to be flush with the shortcuts

|Designs|Right Click Menu|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/150139997-f3e4eeb4-a9b7-4bd3-86ec-05282e09aef5.png)|![image](https://user-images.githubusercontent.com/22185279/150140028-85c097c5-45d3-4248-8a6c-a7eb28252857.png)|

Reply:
> The chevron should to be flush with the shortcuts

Not sure what you mean with this, could you elaborate? Do you mean that we should combine the shortcuts and chevron into one suffix instead of using shortcuts + suffix?

Reply:
Oh sorry, what I mean by that is that the right edge of both the chevron and the shortcuts should be 15px from the edge of the context menu:
![image](https://user-images.githubusercontent.com/22185279/150152322-af3a2758-8f6c-43e2-8982-f154b6c08dc7.png)

(really 16px since we round to 4s in the design system)

Reply:
Ah, okay, now I understand. Yep, I think there are inconsistencies currently in Figma, it has always been an even number at least. Will adjust, thanks!